### ISSUE TITLE
TSC 1.8.9 - TsConfig.json - Files not included by default

### ISSUE THREAD

**Issue by snerks** [Microsoft/TypeScript#8145]
**TypeScript Version:** 

1.8.9

**Windows Version:** 

7 SP1

**Code**

```json
// tsconfig.json
// minimal content, implies all TS/TSX files in same folder and subfolders to be included in the project context?
{}
```

```ts
// A self-contained demonstration of the problem follows...
// HelloWorld.ts
class Startup {
    public static main(): number {
        console.log("Hello World");
        return 0;
    }
}

Startup.main();
```

**Expected behavior:**
Invocation of TSC compiler at the command-line creates a file called HelloWorld.js.
```json
tsc -p .
```
**Actual behavior:** 
Invocation of command-line "TSC -p ." does not create a file.

**Comment by mhegazy** [comment]
```cmd
c:\test\8145>tsc --version
Version 1.8.10

c:\test\8145>type tsconfig.json
// tsconfig.json
// minimal content, implies all TS/TSX files in same folder and subfolders to be included in the project context?
{}

c:\test\8145>type main.ts
// A self-contained demonstration of the problem follows...
// HelloWorld.ts
class Startup {
    public static main(): number {
        console.log("Hello World");
        return 0;
    }
}

Startup.main();

c:\test\8145>dir main.js
 Volume in drive C is OSDisk
 Volume Serial Number is 9E6D-383F

 Directory of c:\test\8145

File Not Found

c:\test\8145>tsc -p .

c:\test\8145>type main.js
// A self-contained demonstration of the problem follows...
// HelloWorld.ts
var Startup = (function () {
    function Startup() {
    }
    Startup.main = function () {
        console.log("Hello World");
        return 0;
    };
    return Startup;
}());
Startup.main();

```

**Comment by mhegazy** [comment]
could not get a repro for this locally. please reopen and provide more information to reproduce the issue.

**Comment by snerks** [comment]
I noticed you are using **tsc v1.8.10**.

I usually use this link to install the Typescript compiler and VS2015 tooling:
[https://www.microsoft.com/en-us/download/details.aspx?id=48593&751be11f-ede8-5a0c-058c-2ee190a24fa6=True](https://www.microsoft.com/en-us/download/details.aspx?id=48593&751be11f-ede8-5a0c-058c-2ee190a24fa6=True)

My globally-installed version is **1.8.9**.
I think this version was installed with the recently-released VS Code 1.0.

As an experiment, I used a local NPM install of **tsc v.1.8.10**.

If i then invoke the compiler thus:

`
.\node_modules\.bin\tsc -p .
`

the HelloWorld.js file is generated.

So, for information:
1) Are you able to test this issue with TSC v1.8.9?
and/or
2) Is there a VS2015 tools installer for TSC v1.8.10 (which seems to resolve the problem) I can use?
